api/lib/domain/events/handle-auto-jury.js

```diff
@@ -89,6 +89,8 @@ async function _autoCompleteUnfinishedTest({
     certificationAssessment.neutralizeUnansweredChallenges();
   }
 
+  certificationAssessment.endByFinalization();
```

Comment:
Je trouve que l'intention fonctionelle n'est pas claire.
Pourquoi pas l'exprimer en disant

```suggestion
  function preventCandidateToAnswerAfterSessionISFinalized(){
    certificationAssessment.endByFinalization();
   }
```

Reply:
ça nous paraissait assez clair, après le renommage en endDueToFinalization ça le sera sûrement plus. Mais je ne vois pas l'apport à encapsuler une méthode explicite d'un objet dans une autre méthode. Soit on fait de l'objet, soit du fonctionnel. Mais je reste ouvert, à voir avec la team

Surtout que le fait que le candidate ne puisse plus répondre, est un effet de bord. Là on veut juste mettre fin à un test non terminé à la finalisation

Reply:
> Là on veut juste mettre fin à un test non terminé à la finalisation
C'est vrai. Mais `endByFinalization()` reste encore mystérieux. Tant pis !